css/store.css

```diff
@@ -0,0 +1,85 @@
+body {
+    font-family: 'Roboto', sans-serif;
+    -webkit-font-smoothing: antialiased;
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  text-align: center;
+  background-image: url(C:\Users\User\Documents\GitHub\HTML-CSS-Module-Project\img\first-background.jpg);
```

Comment:
this path won't resolve correctly; will need to be something like `/img/first-background.jpg`